run.go

```diff
@@ -30,7 +30,7 @@ func (r *Client) Run(ctx context.Context, identifier string, input PredictionInp
 		return nil, err
 	}
 
-	prediction, err = r.Wait(ctx, *prediction, 0, 0)
+	prediction, err = r.Wait(ctx, *prediction, 1, 0)
```

Comment:
`time.Duration` is denominated by nanoseconds, so this should be `1 * time.Second`